src/pages/Playground/index.jsx

```diff
@@ -2,6 +2,7 @@ import MentorCarousel from '../../components/molecules/MentorCarousel/MentorCaro
 import { Body2, Body3, Button, ButtonMeta, Heading1, Heading2, Heading3, Heading4, NavText } from '../../components/shared';
 import Body1 from '../../components/shared/Typography/Body1';
 import { Container } from './styles';
+import Faq from '../../components/molecules/faq/faq';
```

Comment:
Place the FAQ component in the playground for us to check.

Reply:
should I place the faq component like 
  const Playground =()=>{
return (<>
//same as given 
</Container>
<MentorCarousel/>
<Faq/>
</>);
};

is this pseudo code fit for this ?
@Lucif3r-in

Reply:
done @Lucif3r-in